great_expectations/exceptions/exceptions.py

```diff
@@ -48,11 +48,28 @@ class ExpectationSuiteError(DataContextError):
     pass
 
 
-class ExpectationSuiteNotAddedToStoreError(DataContextError):
-    def __init__(self) -> None:
+class ExpectationSuiteNotSavedError(DataContextError):
+    pass
+
+
+class ExpectationSuiteNotAddedToStoreError(ExpectationSuiteError):
```

Comment:
I'd remove `ToStore` from all these exception names to keep that implementation naming detail out of public errors.